internal/devserver/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "@bazel/typescript-devserver",
+    "version": "0.0.0",
+    "license": "Apache-2.0",
+    "devDependencies": {
+        "livereload": "0.6.x",
```

Comment:
since we don't have a lockfile, let's pin these to exact versions

Reply:
done